event_routing_backends/models.py

```diff
@@ -40,6 +43,80 @@ def get_value_from_dotted_path(dict_obj, dotted_key):
     return result
 
 
+class RouterConfigurationModelManager(models.Manager):
```

Comment:
can we rename it to `RouterConfigurationManager`?

Reply:
Done